### DISCUSSION TITLE
Announcing Sionna v0.14

### DISCUSSION THREAD

**Discussion by faycalaa** [NVlabs/sionna#105] [Announcements]
We are happy to announce the Sionna v0.14.0 release:

# Features

## Ray Tracing

- Scene 
       -  Loading of geometries and EM-properties
       - Support to load scenes from external tools
       - Built-in scenes
       - Differentiable computation of propagation paths
       - Differentiable computation of coverage maps
       - 3D interactive viewer in notebooks
       - Ray tracing-based rendering of scenes
- Radio Materials
       - Customization with frequency-dependent properties
       - Integrated ITU radio materials
- Radio Devices
       - Support for arbitrary positions and orientations
       - Support for arbitrary antenna arrays and patterns
- Layer to compute channel impulse responses from propagation paths
- Cameras for visualization of scenes from arbitrary viewpoints
- Utility functions for 3D geometry
 
## Tutorials
- [Sionna Ray Tracing Tutorial notebook](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html)
- [“Primer on Electromagnetics”](https://nvlabs.github.io/sionna/em_primer.html) explaining the theory behind ray tracing
- [Video](https://www.youtube.com/watch?v=7xHLDxUaQ7c]) showing how import real-world environments in Sionna RT using Blender and OpenStreetMap
 
# Fixes
- Fixes issue (#103)
- Fixes broken links in 5G NR PUSCH tutorial
- Updated DOCKERFILE (TF 2.11 as default)
- Updated file headers
     
# Other
- Updated “Made with Sionna” section
- Updated installation guide
- New requirements: `Mitsuba`, `pythreejs`, `ipywidgets`

**Comment by nbecker** [comment]
On fedora 37, I got
ImportError: jit_init_thread_state(): the LLVM backend is inactive because the LLVM shared library ("libLLVM.so") could not be found! Set the DRJIT_LIBLLVM_PATH environment variable to specify its path.
I then did 
sudo dnf install llvm-devel
which fixed that

**Reply by pablosreyero** [reply]
Hey!

I get the same error, but unlike you, I was not able to sort it out. I'm running the code in a Virtual Machine with Ubuntu 23.04, I have anaconda installed and I created an environment. In order to try to solve this problem I executed the command you wrote (`sudo install llvm-devel`), in my terminal (inside the environment where I'm running my code and where all Sionna packages are installed) but I don't get any feedback at all, so I suppose that I installed the llvm-dev package correctly. After running the command in the terminal I launch JupyterNotebook and run the code, but I still get the error. 

Since you were able to solve the problem, is there anything I'm missing? Are there any other ways to overcome this bug? Because it is a pain in the ass. Any help will be appreciated.

Thanks in advance and take care!
